### PR TITLE
PROGRAMMES-5874 Istats counter name fix differences with PAL in AWS

### PR DESCRIPTION
**Pull request by tatitati**
Jira: 
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5874

This ticket fix the problem of the ticket. 
Also cover rest of cases with counter name (which after comparing with PAL I saw they work properly).

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: tests/ValueObject/AnalyticsCounterNameTest.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+     * @dataProvider urlDefaultProvider
+     */
+    public function testCounterNameIsBuildByDefaultInOtherCases(string $expectedValueOutput, string $relativePathProvided)
+    {
+        $analyticsCounterName = new AnalyticsCounterName(null, $relativePathProvided);
+
+        $this->assertEquals($expectedValueOutput, (string) $analyticsCounterName);
+    }
+
+    public function urlDefaultProvider(): array
```
Comment (votemike) [inline tests/ValueObject/AnalyticsCounterNameTest.php]: Do we need a provider with one item in it?

File: tests/ValueObject/AnalyticsCounterNameTest.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            ]),
+
+        ]);
+
+        $analyticsCounterName = new AnalyticsCounterName($programmeContext, $relativePathProvided);
+
+        $this->assertEquals($expectedValueOutput, (string) $analyticsCounterName);
+    }
+
+    public function relativePathForProgrammeWithparentsProvider(): array
```
Comment (votemike) [inline tests/ValueObject/AnalyticsCounterNameTest.php]: Do we need a provider with one item in it?

Comment (tatitati) [reply tests/ValueObject/AnalyticsCounterNameTest.php]: yeah, I should remove this provider. I don't expect to add more scenarios for this case

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**